tests/AgentTests/SchemaDump.hs

```diff
@@ -1,30 +1,63 @@
+{-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE OverloadedStrings #-}
 
 module AgentTests.SchemaDump where
 
 import Control.DeepSeq
 import Control.Monad (void)
+import Data.List (dropWhileEnd)
+import Data.Maybe (fromJust, isJust)
 import Simplex.Messaging.Agent.Store.SQLite
+import Simplex.Messaging.Agent.Store.SQLite.Migrations (Migration (..), MigrationsToRun (..), toDownMigration)
 import qualified Simplex.Messaging.Agent.Store.SQLite.Migrations as Migrations
+import System.Directory (removeFile)
 import System.Process (readCreateProcess, shell)
 import Test.Hspec
 
 testDB :: FilePath
 testDB = "tests/tmp/test_agent_schema.db"
 
-schema :: FilePath
-schema = "src/Simplex/Messaging/Agent/Store/SQLite/Migrations/agent_schema.sql"
+appSchema :: FilePath
+appSchema = "src/Simplex/Messaging/Agent/Store/SQLite/Migrations/agent_schema.sql"
+
+testSchema :: FilePath
+testSchema = "tests/tmp/test_agent_schema.sql"
 
 schemaDumpTest :: Spec
-schemaDumpTest =
+schemaDumpTest = do
   it "verify and overwrite schema dump" testVerifySchemaDump
+  it "verify schema down migrations" testSchemaMigrations
 
 testVerifySchemaDump :: IO ()
 testVerifySchemaDump = do
-  void $ createSQLiteStore testDB "" Migrations.app MCConsole
-  void $ readCreateProcess (shell $ "touch " <> schema) ""
-  savedSchema <- readFile schema
+  savedSchema <- readFile appSchema
```

Comment:
I think this test created the schema file if it didn't exist, now it seems it would fail